Makefile

```diff
@@ -16,97 +16,103 @@ METADATA_DEFAULTS_FILE=${MAKEFILE_PATH}/pkg/config/defaults/aemm-metadata-defaul
-	docker push ${IMG_W_TAG}
+helm-lint-test:
+	${MAKEFILE_PATH}/test/helm/chart-test.sh -l
 
-build-docker-images:
-	${MAKEFILE_PATH}/scripts/build-docker-images -d -p ${SUPPORTED_PLATFORMS} -r ${IMG} -v ${VERSION}
+helm-app-version-test:
+	${MAKEFILE_PATH}/test/helm/helm-app-version-test.sh
 
-push-docker-images:
-	@echo ${DOCKERHUB_TOKEN} | docker login -u ${DOCKERHUB_USERNAME} --password-stdin
-	${MAKEFILE_PATH}/scripts/push-docker-images -p ${SUPPORTED_PLATFORMS} -r ${IMG} -v ${VERSION} -m
+helm-e2e-test:
+	${MAKEFILE_PATH}/test/helm/chart-test.sh
 
-version:
-	@echo ${VERSION}
+license-test:
+	${MAKEFILE_PATH}/test/license-test/run-license-test.sh
 
-latest-tag:
-	@echo ${LATEST_TAG}
+go-report-card-test:
+	${MAKEFILE_PATH}/test/go-report-card-test/run-report-card-test.sh
 
-image:
-	@echo ${IMG_W_TAG}
+test: unit-test e2e-test helm-app-version-test helm-e2e-test license-test go-report-card-test
 
-upload-resources-to-github:
-	${MAKEFILE_PATH}/scripts/upload-resources-to-github
+build-binaries:
+	${MAKEFILE_PATH}/scripts/build-binaries -d -p ${SUPPORTED_PLATFORMS} -v ${VERSION}
 
 generate-k8s-yaml:
 	${MAKEFILE_PATH}/scripts/generate-k8s-yaml
 
-sync-readme-to-dockerhub:
-	${MAKEFILE_PATH}/scripts/sync-readme-to-dockerhub
-
-unit-test: create-build-dir
-	go test ${MAKEFILE_PATH}/... -v -coverprofile=coverage.txt -covermode=atomic -outputdir=${BUILD_DIR_PATH}
-
-e2e-test: build
-	${MAKEFILE_PATH}/test/e2e/run-tests
+gen-helm-chart-archives:
+	${MAKEFILE_PATH}/scripts/generate-helm-chart-archives
 
-validate-json:
-	${MAKEFILE_PATH}/test/json-validator
+upload-resources-to-github:
+	${MAKEFILE_PATH}/scripts/upload-resources-to-github
 
-validate-readme:
-	${MAKEFILE_PATH}/test/readme-validator
+build-release-assets: create-build-dir build-binaries generate-k8s-yaml gen-helm-chart-archives
 
-helm-lint-test:
-	${MAKEFILE_PATH}/test/helm/chart-test.sh -l
+release: build-release-assets upload-resources-to-github
 
-helm-e2e-test:
-	${MAKEFILE_PATH}/test/helm/chart-test.sh
+build-docker-images:
+	${MAKEFILE_PATH}/scripts/build-docker-images -d -p ${SUPPORTED_PLATFORMS} -r ${IMG} -v ${VERSION}
 
-helm-app-version-test:
-	${MAKEFILE_PATH}/test/helm/helm-app-version-test.sh
+push-docker-images:
+	@echo ${DOCKERHUB_TOKEN} | docker login -u ${DOCKERHUB_USERNAME} --password-stdin
+	${MAKEFILE_PATH}/scripts/push-docker-images -p ${SUPPORTED_PLATFORMS} -r ${IMG} -v ${VERSION} -m
 
-helm-tests:
-	helm-app-version-test helm-e2e-test
+sync-readme-to-dockerhub:
+	${MAKEFILE_PATH}/scripts/sync-readme-to-dockerhub
 
-gen-helm-chart-archives:
-	${MAKEFILE_PATH}/scripts/generate-helm-chart-archives
+release-docker: build-docker-images push-docker-images sync-readme-to-dockerhub
 
-license-test:
-	${MAKEFILE_PATH}/test/license-test/run-license-test.sh
 
-go-report-card-test:
-	${MAKEFILE_PATH}/test/go-report-card-test/run-report-card-test.sh
+# Targets intended for local use 
+fmt:
+	goimports -w ./
```

Comment:
I know you didn't change this... but can you make this do a code simplification (`-s`) too.  Go report card will report warnings for code that can use syntax sugar to simplify. 

```
fmt:
   goimports -w ./ && gofmt -s -w ./
```